docs/contributing/coding-standards/js.md

```diff
@@ -133,6 +133,40 @@ Avoid using namespace imports (`import * as namespace`) in code bundled for Comm
 
 Prefer named exports over default exports to avoid compatibility issues with transpiler "synthetic default" as discussed in: https://github.com/alphagov/govuk-frontend/issues/2829
 
+## Throwing errors
```

Comment:
This is just a starter for 10 listing the spirit in which the errors have been implemented in this PR. Happy to discuss and update.